### ISSUE TITLE
(tdweb): ReferenceError: window is not defined

### ISSUE THREAD

**Issue by esindger** [tdlib/td#741]
Hello,
it's unnable to work with files from another **web worker**, `tdweb` never respond. Library just falling down without any error. I found out it's happening because of the following [peace of code](https://github.com/tdlib/td/blob/master/example/web/tdweb/src/index.js#L445):

```js
const request = window.indexedDB.open(this.instanceName);
```
Web workers are also supporting indexedDB, so it's unnecessary to restrict environment.

**Comment by levlam** [comment]
Thanks. This looks reasonable.

**Comment by UltimateZero** [comment]
I'm having the same issue.

Replacing `window` with `self` should make it work within both main thread and web workers:
```javascript
const request = window.indexedDB.open(this.instanceName);
```
to
```javascript
const request = self.indexedDB.open(this.instanceName);
```

**Comment by levlam** [comment]
This was fixed in tdweb 1.6.0.
